installing/installing-fips.adoc

```diff
@@ -14,7 +14,9 @@ For more information about the NIST validation program, see link:https://csrc.ni
 
 [IMPORTANT]
 ====
-To enable FIPS mode for your cluster, you must run the installation program from a {op-system-base} computer configured to operate in FIPS mode. For more information about configuring FIPS mode on RHEL, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/security_hardening/assembly_installing-the-system-in-fips-mode_security-hardening[Installing the system in FIPS mode].
+To enable FIPS mode for your cluster, you must run the installation program from a {op-system-base} computer configured to operate in FIPS mode. Running {op-system-base} 9 to install an {product-title} cluster is not supported.
```

Comment:
```suggestion
To enable FIPS mode for your cluster, you must run the installation program from a {op-system-base} 8 computer configured to operate in FIPS mode. Running {op-system-base} 9 with FIPS mode enabled to install an {product-title} cluster is not possible.
```

Reply:
Thanks, @zaneb . I updated the content as per your suggestion. Would you be OK to leave a LGTM comment on the PR?